### ISSUE TITLE
Problems with RCS

### ISSUE THREAD

**Issue by Mauladen** [Wando1423/Antario#48]
RCS works normally only when it is always on.
If you enable it only when you work with Aimbot, then there will not be any compensation at all
You can see for yourself if you test

**Comment by exp111** [comment]
Yea I had that problem too. If nobody fixes it till I'm home I'll search for the problem

**Comment by Mauladen** [comment]
I will be very grateful to you if you fix this bug

**Comment by Wando1423** [comment]
Ok, made some changes. Check now.

**Comment by Mauladen** [comment]
Nothing changed

**Comment by Wando1423** [comment]
If you'd have some time, could you check if its working on standard aimtux? If it is, its a problem with my fork and if its not, then a new update broke it.

**Comment by Mauladen** [comment]
This bug is also in AimTux, for a long time already

**Comment by Wando1423** [comment]
Ok, so I will try to fix it in 10 hours (when i'll be home and have a possibility to test).

**Comment by Mauladen** [comment]
Good
Thank you!

**Comment by exp111** [comment]
Dunno had this bug yesterday... but can't figure out how to get it now. RCS works perfectly fine with always on and without too.

**Comment by Wando1423** [comment]
Looks like it sometimes works and sometimes it doesnt.
Check with Kill timeout / target lock. I heard they are the reason.

**Comment by Mauladen** [comment]
Yes, it does not always work, I can throw off your config and you will test with my settings

Here is a link to my config --> [Download](http://rgho.st/private/8RHps45dZ/266a3dcb8af6c6a60e6a83a1ddfc502a)

**Comment by exp111** [comment]
Tried your config: I think it's because of the low fov. It's so low (1.3) that after shooting the aimbot thinks it has lost its target. Maybe let the recoil control check if the recoilcrosshair is on the target.
Temporary solution: Higher fov or just just always on

**Comment by Mauladen** [comment]
I played private cheats with other games and there was no such problem
Even with such a small FOV everything worked perfectly

**Comment by Wando1423** [comment]
Well, it should work, but yeah, if you lose the target, it might brake.
Ill test it on the weekend and look for  a possible fix.

**Comment by Mauladen** [comment]
Well, I would like to know how to contact you (you are in the Telegram?)
